de.js

```diff
@@ -0,0 +1,334 @@
+    seed_phrase_key: `Ihre Seed-Phrase ist der Schlüssel zu Ihrem Wallet. Jeder mit dieser Phrase kann auf Ihr Guthaben zugreifen. Tippen Sie hier, um mehr zu erfahren`,
+    written_down: `Ich habe es aufgeschrieben`,
+    screenshot_alert: `Screenshots oder andere digitale Kopien Ihrer Seed Phrase könnten an Dritte weitergegeben werden.`,
+    seed_phrase_access: `Jeder mit Ihrer Seed Phrase kann auf Ihr Guthaben zugreifen. Geben Sie Ihre Seed Phrase nicht an Dritte weiter. MyDoge speichert Ihre Seed Phrase nicht für Sie. Sollte Ihr Gerät beschädigt, verloren oder gestohlen werden oder anderweitig unzugänglich sein, ist Ihre Seed Phrase die einzige Möglichkeit, Ihr Wallet wiederherzustellen.`,
+    icloud_backup: `iCloud-Backup`,
+    cloud_backup: `Cloud-Backup`,
+    dont_lose_wallet: `Verlieren Sie nicht Ihr Portemonnaie!`,
+    save_online_backup: `Speichern Sie ein Online-Backup in`,
+    icloud: `iCloud`,
+    google_drive: `Google Drive`,
+    do_this_later: `Ich mache das später`,
+    confirm_seed_phrase: `Seed-Phrase bestätigen`,
+    select_correct_word: `Bitte wählen Sie aus der Liste unten an jeder Stelle das richtige Wort aus.`,
+    enter_seed_phrase: `Seed-Phrase eingeben`,
+    enter_seed_phrase_instructions: `Geben Sie Ihre 12-Wort-Seed-Phrase ein, um Ihr bestehendes Wallet wiederherzustellen`,
+    paste: `Paste`,
+    restore_wallet: `Wallet wiederherstellen`,
+    wallet_imported: `Geldbörse importiert!`,
+    wallet_imported_body: `Ihr Portemonnaie ist einsatzbereit.`,
+    restore_existing_wallet: `Stellen Sie Ihr vorhandenes Wallet wieder her`,
+    restore_existing_wallet_instructions: `Du hast also schon eine Brieftasche? Dann lass es uns wieder angehen!`,
+    restore_from: `Wiederherstellen von`,
+    restore_from_cloud_description: `Stellen Sie Ihr Wallet wieder her von Ihrem`,
+    restore_with_seed_phrase: `Wiederherstellen mit einer Seed-Phrase`,
+    restore_with_seed_phrase_description: `Verwenden Sie Ihre Seed-Phrase, um ein vorhandenes MyDoge-Wallet wiederherzustellen`,
+    no_cloud_backup: `Kein Cloud-Backup`,
+    no_cloud_backup_body: `Cloud-Backup nicht gefunden. Bitte stellen Sie es aus einer Seed-Phrase wieder her oder erstellen Sie ein neues Wallet`,
+    create_wallet: `Erstellen Sie eine Brieftasche`,
+    already_have_wallet: `Ich habe bereits eine Brieftasche`,
+    buy_send_receive_dogecoin: `Dogecoin kaufen, senden und empfangen`,
+    send_doge_tips: `Senden Sie Doge-Tipps zu 𝕏, bewahren Sie Ihre Doge sicher auf und seien Sie Ihre eigene Bank`,
+    shop_gift_cards_apparel: `Geschenkkarten und Bekleidung kaufen`,
+    buy_gift_cards: `Kaufen Sie Geschenkkarten mit Dogecoin oder shoppen Sie die neueste Doge Fashion`,
+    enter_the_dogeverse: `Betreten Sie das Dogeverse`,
+    claim_dogtag: `Fordern Sie Ihren DogeTag an, finden Sie Freunde, erhalten Sie Zahlungen mit Ihrem Link und teilen Sie Memes`,
+    so_much_tip: `Also Tipp, viel Wow!`,
```

Comment:
```suggestion
    so_much_tip: `Viel Trinkgeld, viel Wow!`,
```
I'm not sure about this translation. I've read the English text but what's the meaning?